pom.xml

```diff
@@ -690,6 +690,16 @@
                 <artifactId>javax.el-api</artifactId>
                 <version>3.0.0</version>
             </dependency>
+            <dependency>
+                <groupId>com.squareup.retrofit</groupId>
+                <artifactId>retrofit</artifactId>
+                <version>2.0.0-beta2</version>
```

Comment:
The syntax has changed very much between 1.x and 2.0 and from what I've seen the 2.0 release is being prepared and supposed to be released soon, hopefully before our 2.0 code freeze. :)